.travis.yml

```diff
@@ -9,11 +9,11 @@ services:
 
 # Disable automatic submodule fetching ...
 git:
+    depth: 9999999
```

Comment:
Can you point me to an example of using makd in this context?  (Note that the project already has its own makefiles.)